metrics/web/lib/common/presentation/button/widgets/metrics_button.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:metrics/base/presentation/widgets/hand_cursor.dart';
+import 'package:metrics/common/presentation/button/theme/attention_level/metrics_button_attention_level.dart';
+import 'package:metrics/common/presentation/metrics_theme/model/metrics_theme_data.dart';
+import 'package:metrics/common/presentation/button/theme/style/metrics_button_style.dart';
+import 'package:metrics/common/presentation/metrics_theme/widgets/metrics_theme.dart';
+
+/// An abstract widget that displays a [RaisedButton] and
```

Comment:
```suggestion
/// An abstract widget that displays a metrics styled button and
```
It might be good to avoid details here as we want to incapsulate details.